app/services/provider_authorisation.rb

```diff
@@ -145,22 +149,22 @@ def full_authorisation?(permission:, course:)
       if @actor.providers.include?(ratifying_provider) && @actor.providers.include?(training_provider)
         # If not, there is something wrong with the setup
         if [training_provider_can, ratifying_provider_can].none?
-          errors << :requires_training_provider_permission
-          errors << :requires_ratifying_provider_permission
+          add_error(permission, :requires_training_provider_permission)
+          add_error(permission, :requires_ratifying_provider_permission)
```

Comment:
It is possible to have an error messge combining these two errors? e.g. `:requires_ratifying_or_training_provider_permission`

Reply:
:+1: https://github.com/DFE-Digital/apply-for-teacher-training/pull/4211/commits/204c7a2dcbf27493229ac93ce8a07f8b38e8851a